docs/_getting-started-macos-android.md

```diff
@@ -69,21 +69,30 @@ Next, select the "SDK Tools" tab and check the box next to "Show Package Details
 
 Finally, click "Apply" to download and install the Android SDK and related build tools.
 
-<h4>3. Configure the ANDROID_SDK_ROOT environment variable</h4>
+You can also run the following command after setting ANDROID_HOME.
+
+```shell
+sdkmanager "platforms;android-30" "system-images;android-30;default;x86_64" "system-images;android-30;google_apis;x86"
+sdkmanager "cmdline-tools;latest" "build-tools;30.0.2"
+```
```

Comment:
I would refrain from mentioning those commands as they require you to install the CLI tools which we're not asking the users to do IIRC on Mac